src/Silex/EventListener/MiddlewareListener.php

```diff
@@ -75,7 +75,7 @@ public function onKernelResponse(FilterResponseEvent $event)
             return;
         }
 
-        foreach ((array) $route->getOption('_after_middlewares') as $callback) {
+        foreach ($route->getOption('_after_middlewares') ?: array() as $callback) {
```

Comment:
SplPriorityQueue implements Iterator, but cannot be cast to array.

Prior to this patch, `$route->getOption('_before_middlewares')` could return `null` or `array`. After this patch, it can return `null` or `SplPriorityQueue`.

I changed it, so that in the `null` case, the expression results in `array()`, which is the same as it was with the cast. But in the `SplPriorityQueue` case, the expression results in `SplPriorityQueue`.